tekton/README.md

```diff
@@ -59,14 +89,14 @@ To use [`tkn`](https://github.com/tektoncd/cli) to run the
    apiVersion: tekton.dev/v1alpha1
    kind: PipelineResource
    metadata:
-     name: tekton-triggers-vX-Y-
+   name: tekton-triggers-git
    spec:
-     type: git
-     params:
-       - name: url
-         value: https://github.com/tektoncd/triggers
-       - name: revision
-         value: vX.Y.Z-invalid-tags-boouuhhh # REPLACE with the commit you'd like to build from
+   type: git
+   params:
+   - name: url
+      value: https://github.com/tektoncd/triggers
+   - name: revision
+      value: master  # REPLACE with the commit you want to release
```

Comment:
same as above; might be useful to set to invalid value to prevent unintended copy paste.